source/release-notes.txt

```diff
@@ -13,6 +13,28 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.46.2
+----------------
+
+*Released May 9, 2025*
+
+New Features:
+
+- Allow users to disable dbStats and collStats in application preferences 
+  (:issue:`COMPASS-5387`)
+- Add an error modal for viewing document validation error details 
+  (:issue:`COMPASS-8868`)
+- Use an iterable cursor for schema analysis (:issue:`COMPASS-9150`, :issue:`COMPASS-9315`)
+
+
+Bug Fixes:
+
+- Disable spellcheck before creating the window to prevent network traffic in
```

Comment:
[nitpick] "Disable spellcheck to prevent network traffic in isolated mode" is a little clearer